build.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+check_dependancy() {
```

Comment:
*dependency

Reply:
You can spell it with an 'a' too although according to http://www.dictionary.com/browse/dependancy its fallen out of use. 
I will change.